applications/ExaquteSandboxApplication/custom_processes/divergence_process.h

```diff
@@ -0,0 +1,217 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+//
+
+#ifndef KRATOS_DIVERGENCE_PROCESS_H
+#define KRATOS_DIVERGENCE_PROCESS_H
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/geometry_utilities.h"
+#include "includes/model_part.h"
+#include "processes/process.h"
+#include "utilities/variable_utils.h"
+#include "exaqute_sandbox_application_variables.h"
```

Comment:
I think you can include this one in the cpp.

Reply:
Done.